prime-router/src/main/kotlin/azure/SettingsFunctions.kt

```diff
@@ -53,9 +53,7 @@ class GetOneOrganization(
         ) request: HttpRequestMessage<String?>,
         @BindingName("organizationName") organizationName: String,
     ): HttpResponseMessage {
-        // Is the API user an Okta Sender?
-        val oktaSender = request.headers["authentication-type"] == "okta"
-        return getOne(request, organizationName, OrganizationAPI::class.java, null, oktaSender)
+        return getOne(request, organizationName, OrganizationAPI::class.java, organizationName)
```

Comment:
There's no reason why any Settings call needs to care whether a user is a DHSender_blahblah" claim or a "DHblahblah" claim.  So I removed all the "oktaSender" stuff from the Settings code.